Makefile

```diff
@@ -262,4 +262,41 @@ go.cachedir:
 go.mod.cachedir:
 	@go env GOMODCACHE
 
-.PHONY: cobertura reviewable submodules fallthrough go.mod.cachedir go.cachedir run crds.clean $(TERRAFORM_PROVIDER_SCHEMA)
+CONCURRENCY ?= 30
+DEP_CONSTRAINT := >= 0.0.0
+ifeq (-,$(findstring -,$(VERSION)))
+    DEP_CONSTRAINT = >= 0.0.0-0
+endif
+load-pkg: $(UP) build.all
+	@$(INFO) Batch processing smaller provider packages for: $(SUBPACKAGES)
+	@for p in $(PLATFORMS); do \
+		mkdir -p "$(XPKG_OUTPUT_DIR)/$$p"; \
+	done
+	@$(UP) xpkg batch --smaller-providers $$(echo -n $(SUBPACKAGES) | tr ' ' ',') \
+		--family-base-image $(BUILD_REGISTRY)/$(PROJECT_NAME) \
+		--platform $(BATCH_PLATFORMS) \
+		--provider-name $(PROJECT_NAME) \
+		--family-package-url-format $(XPKG_REG_ORGS)/%s:$(VERSION) \
+		--package-repo-override monolith=$(PROJECT_NAME) --package-repo-override config=provider-family-$(PROVIDER_NAME) \
+		--provider-bin-root $(OUTPUT_DIR)/bin \
+		--output-dir $(XPKG_OUTPUT_DIR) \
+		--store-packages $$(echo -n $(SUBPACKAGES) | tr ' ' ',') \
+		--build-only=true \
+		--examples-root $(ROOT_DIR)/examples \
+		--examples-group-override monolith=* --examples-group-override config=providerconfig \
+		--auth-ext $(ROOT_DIR)/package/auth.yaml \
+		--crd-root $(ROOT_DIR)/package/crds \
+		--crd-group-override monolith=* --crd-group-override config=$(PROVIDER_NAME) \
+		--package-metadata-template $(ROOT_DIR)/package/crossplane.yaml.tmpl \
+		--template-var XpkgRegOrg=$(XPKG_REG_ORGS) --template-var DepConstraint="$(DEP_CONSTRAINT)" --template-var ProviderName=$(PROVIDER_NAME) \
+		--concurrency $(CONCURRENCY) \
+		--push-retry 10 || $(FAIL)
+	@$(OK) Done processing smaller provider packages for: $(SUBPACKAGES)
+
+	@for p in $(PLATFORMS); do \
+		for s in $(SUBPACKAGES); do \
+			docker tag $$(docker load -qi $(XPKG_OUTPUT_DIR)/$$p/$(PROJECT_NAME)-$$s-$(VERSION).xpkg | awk -F: '{print $$3}') $(PROJECT_NAME)-$$s-$${p#"linux_"}:$(VERSION); \
```

Comment:
```suggestion
			docker tag $$(docker load -qi $(XPKG_OUTPUT_DIR)/$$p/$(PROJECT_NAME)-$$s-$(VERSION).xpkg | awk -F: '{print $$3}') 
 $(XPKG_REG_ORGS)/$(PROJECT_NAME)-$$s-$${p#"linux_"}:$(VERSION); \
```

Reply:
After making this change, when we run the `make load-pkg SUBPACKAGES="cloudplatform config" XPKG_REG_ORGS="turkenf"` code, we get the following images as we want:
```
REPOSITORY                                 TAG                         IMAGE ID       CREATED         SIZE
turkenf/provider-gcp-cloudplatform-arm64   v0.36.0-rc.0.11.g3763f359   9ace1942f448   5 hours ago     218MB
turkenf/provider-gcp-config-arm64          v0.36.0-rc.0.11.g3763f359   d6479a95b58b   5 hours ago     217MB
turkenf/provider-gcp-cloudplatform-amd64   v0.36.0-rc.0.11.g3763f359   e83e0cf8a384   5 hours ago     229MB
turkenf/provider-gcp-config-amd64          v0.36.0-rc.0.11.g3763f359   81acd1f7c2bd   5 hours ago     227MB
```
But when we push to the docker hub and use the generated images, provider `HEALTHY=Unknown` remains and I get the following error:

```
Warning  ResolveDependencies  7s (x7 over 69s)     packages/providerrevision.pkg.crossplane.io  cannot resolve package dependencies: missing dependencies: [turkenf/provider-family-gcp]
```

In this case there seems to be two options to continue, either we'll tag and push the image created for the config as `turkenf/provider-family-gcp`(`docker tag turkenf/provider-gcp-config-amd64:v0.36.0-rc.0.11.g3763f359 turkenf/provider-family-gcp:v0.36.0-rc.0.11.g3763f359`) and update the guide accordingly, or continue with the current method.

Reply:
Thanks for checking this @turkenf. We had better handle the name conversion in the `load-pkg` target itself. Otherwise, the users will manually need to retag the loaded family config package themselves. Let's do this for them automatically.